src/utils/OneUI/OneUI.js

```diff
@@ -16,7 +16,7 @@ class OneUI {
      * @param {number} themeConfig.maxTime - Max time to wait for the theme to be loaded
      * @param {Object} themeConfig.ponyfillOptions - Set of options that can be used to configure the ponyfill. Options: https://www.npmjs.com/package/css-vars-ponyfill#options
      */
-    static init({ themeURL = '', maxTime = DEFAULT_LOADING_TIMEOUT, ponyfillOptions } = {}) {
+    static init({ themeURL = '', maxTime = DEFAULT_LOADING_TIMEOUT, ponyfillOptions }) {
```

Comment:
removing this default, would make `OneUI.init()` throw an error, wouldn't it? Probably there should be a test for it, since that's the simplest implementation, but I cannot see any at the moment.

Reply:
Hmm I may have been wrong here. I thought that default was superfluous since the destructured keys have defaults of their own but the default I removed was probably there in case `init` would receive a falsey argument, right?